doc/classes/Semaphore.xml

```diff
@@ -13,13 +13,19 @@
 		<method name="post">
 			<return type="int" enum="Error" />
 			<description>
-				Lowers the [Semaphore], allowing one more thread in. Returns [constant OK] on success, [constant ERR_BUSY] otherwise.
+				Lowers the [Semaphore], allowing one more thread in. Always return [constant OK] for backwards compatibility.
```

Comment:
For this and `wait`, since it mentions backwards compatibility, it raises the question of what's the "new" way to use this API where these return values are no longer relevant.

Reply:
The implementation of these two methods changed, so they always succeed. There is no need to check for return value now.

`master` changed their return type to `void`.

Reply:
Maybe the wording could be changed to something like, "This method internals' can't possibly fail, but an error code is returned for back compat, which will always be OK".